Description: Use $PREFIX/include as include path for ufc
 Upstream is not interested in switching to this path.
Forwarded: not-needed
Author: Min RK <benjaminrk@simula.no>, Johannes Ring <johannr@simula.no>
Last-Update: 2019-05-02

--- ffc-2019.1.0.orig/ffc/backends/ufc/__init__.py
+++ ffc-2019.1.0/ffc/backends/ufc/__init__.py
@@ -62,6 +62,8 @@ from ffc.backends.ufc.form import *
 # Get abspath on import, it can in some cases be
 # a relative path w.r.t. curdir on startup
 _include_path = os.path.dirname(os.path.abspath(__file__))
+import sys
+_include_path = sys.prefix + "/include"
 
 def get_include_path():
     "Return location of UFC header files"
